### PR TITLE
feat: Update menu bar layout

### PR DESCRIPTION
**Pull request by arianeleonard**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [x] Tested on all relevant platforms

## Other information

Update menu bar + jokes and posts small layout changes

## Internal Issue (If applicable):

https://dev.azure.com/nventive/Practice%20committees/_backlogs/backlog/Mobile%20(.Net)/Features/?workitem=238538

### CODE REVIEW

File: src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml [hunk 1]
```diff
@@ -19,7 +21,9 @@
 		<Setter Property="FontFamily"
 				Value="{StaticResource FontFamily01}" />
 		<Setter Property="Padding"
-				Value="20" />
+				Value="20,9.5,20,7.5" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml:24]: How does it look with integer instead of decimal?

**Review by carlh98 — APPROVED** [review]

File: src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml (hunk 1 above)
Comment (arianeleonard) [reply src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml:24]: It look good but I can round up.

Comment (carlh98) [reply src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml:24]: Let's go with that for now

Comment (carlh98) [reply src/app/ApplicationTemplate.UWP/Views/Styles/Controls/BottomTabBarButton.xaml:24]: Let's go with that for now

**Review by carlh98 — APPROVED** [review]

**Pull request merged**